examples/annotated_diffusion.ipynb

```diff
@@ -0,0 +1,1602 @@
+{
```

Comment:
<p>Could you add here: &nbsp;"- which itself is based on the&nbsp;<a href="https://github.com/hojonathanho/diffusion" target="_blank">original TensorFlow implementation</a>." (after Phil Wang's implementation)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>